src/meteodatalab/operators/omega_slope.py

```diff
@@ -42,5 +51,9 @@ def omega_slope(
         / (dak_dz / pc.surface_pressure_ref + dbk_dz)
     ).reduce(cumdiff, dim="z")
 
-    res.attrs = etadot.attrs
-    return res
+    return xr.DataArray(
+        data=res,
+        attrs=metadata.override(
+            etadot.message, discipline=0, parameterCategory=2, parameterNumber=8
```

Comment:
Maybe there could be a comment indicating that this is the "Vertical velocity (pressure)"